sktime/transformations/series/cos.py

```diff
@@ -15,6 +15,8 @@
 class CosineTransformer(_SeriesToSeriesTransformer):
     """Cosine transformation.
 
+    This is a wrapper around numpy's cosine function (see :func:`numpy.cosine`).
```

Comment:
```suggestion
    This is a wrapper around NumPy's cosine function (see :func:`numpy.cos`).
```

Reply:
You can also add this section which will create a link to the numpy documentation: 

```
    See Also
    --------
    numpy.cos
```